### DISCUSSION TITLE
Different ylim and xlim for axes whem making column/row faceted plots

### DISCUSSION THREAD

**Discussion by rsatapat** [pydata/xarray#8165] [Q&A] [answered]
### What is your issue?

I am plotting on multiple axes using the col keyword but I want to be able to pass different arguments (xticks, xlim etc.) for the different axes. There doesn't seem to be a method to do that.

**Comment by mathause** [comment, marked as answer]
I think the workaround for this is to loop over the facetgrid axes (`fg.axs`) after the plot is created and update the limits etc.

**Reply by rsatapat** [reply]
True! That seems to be the case for now. I think it would be a good idea to be able to give a list of the appropriate length for each of these parameters for each axis.
